File: src/main/java/burrow/furniture/standard/RootCommand.java
package burrow.furniture.standard;

import burrow.core.command.Command;
import burrow.core.command.CommandContext;
import burrow.core.command.CommandType;
import picocli.CommandLine;

@CommandLine.Command(name = "root", description = "Display the root directory of the chamber.")
@CommandType(StandardFurniture.COMMAND_TYPE)
public class RootCommand extends Command {
    public RootCommand(final CommandContext commandContext) {
        super(commandContext);
    }

    @Override
    public Integer call() {
        final var rootAbsolutePath = use(StandardFurniture.class).getRootAbsolutePath();
        buffer.append(rootAbsolutePath);

        return CommandLine.ExitCode.OK;
    }
}

File: src/main/kotlin/furnishing/standard/RootCommand.kt
package burrow.furnishing.standard

import burrow.kernel.command.Command
import burrow.kernel.command.CommandData
import picocli.CommandLine

@CommandLine.Command(
    name = "root",
    description = ["Displays the path of chamber root directory."]
)
class RootCommand(data: CommandData) : Command(data) {
    override fun call(): Int {
        stdout.println(chamber.rootPath)

        return CommandLine.ExitCode.OK
    }
}